### ISSUE TITLE
Feature Request: Command Lock Mode

### ISSUE THREAD

**Issue by wyatt-webb** [shawn-lindberg/sandman#18]
Sometimes there are false positive wake up events and command execution which can cause the bed to move when it isn't intended. A feature in which the user can put Sandman into a locked mode so limited commands can be interpreted may be helpful. For instance, in the locked mode the user can only issue an unlock or status command.

**Comment by shawn-lindberg** [comment]
I agree that this sounds like a good idea. This might be a good thing to implement at roughly the same time as "quiet mode".
